core/src/test/scala/io/finch/FinchSpec.scala

```diff
@@ -266,7 +266,7 @@ trait FinchSpec extends FlatSpec
       val resultX = await(x(input))
       val resultY = await(y(input))
 
-      Eq[Option[(Input, Try[Output[A]])]].eqv(resultX, resultY)
+      Eq[Option[(Input, Either[Throwable, Output[A]])]].eqv(resultX, resultY)
```

Comment:
@vkostyukov 

This guy is failing to find an implicit value here, I'm not sure if I can import from somewhere? Please let me know if you have any idea. 

```scala
[error] /Users/prayagupd/buybest/finch/core/src/test/scala/io/finch/FinchSpec.scala:269:9: could not find implicit value for parameter ev: cats.kernel.Eq[Option[(io.finch.Input, Either[Throwable,io.finch.Output[A]])]]
[error]       Eq[Option[(Input, Either[Throwable, Output[A]])]].eqv(resultX, resultY)
[error]         ^
```

Reply:
Maybe instance for `Eq[Throwable]` is missing?

Reply:
You can probably try defining it under `io.finch.MissingInstances`.

Reply:
Ahh I see. Let me try that. 

Also, I rebased from cats-effect yesterday with more `twitter.Try` to `Either[Throwable, A]`, just push few minutes ago. Please let me know if you see anything.

Reply:
Yeah, once CI is happy I'm happy.

Reply:
figured out that scala 2.11.x did not have `.map`/ `.flatMap` in it - https://www.scala-lang.org/api/2.11.8/#scala.util.Either, finch still supports `2.11.x` i believe :)

Reply:
Yeah, or just use `either.right.map`.

Reply:
`.right` looks deprecated in 2.13.x, so i think sticking with pattern match would be nice, 

```scala
scala> data.right
            ^
       warning: method right in class Either is deprecated (since 2.13.0): Either is now right-biased
res1: scala.util.Either.RightProjection[String,Int] = RightProjection(Left(error))
```